scripts/tasks/src/lint-imports.ts

```diff
@@ -200,17 +201,15 @@ function lintImports(
     }
   }
 
-  /**
-   * @param importMatch - Result of running `importStatementRegex` against a single import
-   * (`[1]` will be the import path)
-   */
-  function _evaluateImport(
-    filePath: string,
-    importMatch: RegExpMatchArray,
-    importErrors: ImportErrors,
-    isExample?: boolean,
-  ) {
-    const importPath = importMatch[2];
+  // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
no-shadow is more annoying than helpful in ESM environments , needed to disable as there is already `options` in the file ( not in the scope though... )